app/Repositories/Playlist/PlaylistRepository.php

```diff
@@ -26,11 +33,11 @@ public function __construct(Playlist $model)
      * @param Project $project
      * @return int
      */
-    public function getOrder(Project $project)
+    public function getOrder(Project $project) 
```

Comment:
Public functions such as getOrder, saveList, clone, must also be defined in the PlaylistRepositoryInterface for consistency.